code/game/objects/items/stunbaton.dm

```diff
@@ -266,9 +266,7 @@
 	if(!target.IsKnockdown())
 		to_chat(target, span_warning("Your muscles seize, making you collapse[trait_check ? ", but your body quickly recovers..." : "!"]"))
```

Comment:
Shouldn't this flavour text be updated?

Reply:
I don't think so

Reply:
But it's not making you collapse anymore?

Reply:
It amkes you collapse but you recover so fast that you don't even get knockdowned